src/index.tsx

```diff
@@ -22,58 +22,61 @@ WebFont.load({
   },
 });
 
-const config = getConfig();
```

Comment:
I had to split up the root component so `react-router` would have the `BrowserRouter` available in context for programmatic navigation after authentication.